pkg/controller/actuator.go

```diff
@@ -134,14 +134,18 @@ func (a *actuator) Reconcile(ctx context.Context, log logr.Logger, ex *extension
 
 	istioNamespace, istioLabels, err := a.findIstioNamespaceForExtension(ctx, ex)
 	if err != nil {
+		// we do not reconcile hibernated clusters, as they don't have a Gateway
+		// resource for the extension to get the istio namespace from
+		if cluster.Shoot != nil && cluster.Shoot.Spec.Hibernation != nil &&
```

Comment:
Can we use this https://github.com/gardener/gardener/blob/master/extensions/pkg/controller/shoot.go#L58

Reply:
yes